src/app/showcase/fonts-showcase/fonts-showcase.component.html

```diff
@@ -0,0 +1,55 @@
+<div class="fonts">
```

Comment:
Synes vi bør vise fonts-examples som html i html-vieweren, så man som udvikler kan se hvordan man indsætter det. Det betyder ikke, at vi behøver vise font-examples som "resultat", som vi gør i de andre showcase komponenter.

Reply:
Agreed - in next branch it will be, yes.